context/app/static/js/components/searchPage/SearchTransport.ts

```diff
@@ -0,0 +1,21 @@
+// Adapted from https://github.com/searchkit/searchkit/blob/6f3786657c8afa6990a41acb9f2371c28b2e0986/packages/searchkit/src/core/transport/AxiosESTransport.ts.
```

Comment:
```suggestion
// Adapted from https://github.com/searchkit/searchkit/blob/6f3786657c8afa6990a41acb9f2371c28b2e0986/packages/searchkit/src/core/transport/AxiosESTransport.ts
```

The dot at the end made the link 404 when I clicked through